spec/support/rails_template.rb

```diff
@@ -79,6 +79,19 @@
   append_file 'app/assets/javascripts/active_admin.js', "//= require turbolinks\n"
 end
 
+# Setup webpacker if necessary
+if ENV["BUNDLE_GEMFILE"] == File.expand_path("../../gemfiles/rails_60_webpacker.gemfile", __dir__)
+  inject_into_file 'config/initializers/active_admin.rb', "\n  config.use_webpacker = true\n", before: /^end/
+  rake "webpacker:install"
+  create_file 'app/javascript/packs/active_admin.scss'
+  create_file 'app/javascript/packs/active_admin/print.scss'
+  create_file 'app/javascript/packs/active_admin.js'
+  append_file 'app/javascript/packs/active_admin.js', "import './active_admin.css';"
+  gsub_file 'config/webpacker.yml', /^.*extract_css.*$/, <<-YML
+  extract_css: true
+  YML
```

Comment:
Use `gsub_file 'config/webpacker.yml', /^.*extract_css.*$/, "\n  extract_css: true"` for consistency with other `gsub_file` in this generator.